src/answers-umd.js

```diff
@@ -273,20 +273,22 @@ class Answers {
     if (parsedConfig.onStateChange && typeof parsedConfig.onStateChange === 'function') {
       parsedConfig.onStateChange(
         Object.fromEntries(storage.getAll()),
-        this.coreAdapter.storage.getCurrentStateUrlMerged());
+        this.core.storage.getCurrentStateUrlMerged());
     }
 
     this.components
-      .setCore(this.coreAdapter)
+      .setCore(this.core)
       .setRenderer(this.renderer);
 
     this._setDefaultInitialSearch(parsedConfig.search);
 
+    this.core.init();
```

Comment:
Why did we move this out?

Reply:
Core is no longer an async dependency. I can rever this though. This is better put in a different PR